models/DataObject/ClassDefinition/Data/AdvancedManyToManyRelation.php

```diff
@@ -24,7 +24,7 @@
 use Pimcore\Model\Document;
 use Pimcore\Model\Element;
 
-class AdvancedManyToManyRelation extends ManyToManyObjectRelation implements CustomResourcePersistingInterface, QueryResourcePersistenceAwareInterface
```

Comment:
That was already fixed in 2f7e18f5e76d31e657a3cb83f39e6b0cff04c1bb 
Can you please take that away? (also for `AdvancedManyToManyObjectRelation` 
Thanks!

Reply:
I rebased to master